bedrock/firefox/templates/firefox/browsers/compare/index.html

```diff
@@ -21,10 +21,6 @@
   {% include 'firefox/browsers/compare/includes/subnav.html' %}
 {% endblock %}
 
-{% block page_banner %}
-  {% include 'includes/banners/mobile/mobile-compare.html' %}
```

Comment:
Question: there's a CSS bundle just above here `{{ css_bundle('mobile-compare-banner') }}` which might be able to be removed also?

Reply:
Oh there's a JS bundle called `mobile_compare_banner` which can be removed as well?

Reply:
Thanks for the keen eye!